vendor/bat-native-ads/src/bat/ads/internal/client.cc

```diff
@@ -515,6 +515,25 @@ const std::map<std::string, std::deque<uint64_t>>
   return client_state_->creative_set_history;
 }
 
+void Client::AppendTimestampToConversionHistoryForUuid(
+    const std::string& creative_set_id,
+    const uint64_t timestamp_in_seconds) {
```

Comment:
I would use `const uint64_t&` as we still do x86 builds

Reply:
Do we really want this small performance increase?

Reply:
Sounds good here, just to remember for the feature.